app/components/user-dropdown.js

```diff
@@ -5,16 +5,35 @@ const {
   inject: { service }
 } = Ember;
 
+/**
+  `user-dropdown` composes
```

Comment:
I think should indicate that it's the dropdown menu in the main site navigation. This could potentially use a better naming convention.